xls-workshop-openlane.ipynb

```diff
@@ -162,10 +162,20 @@
         "  n\n",
         "}\n",
         "\n",
+        "\n",
+        "// This is a function to turn the \"adder1\" function into 8-bit input, 8-bit output function\n",
+        "// Refer to \"Bit Slice Expressions\" section in DSLX reference (Bit Slice Expressions: https://google.github.io/xls/dslx_reference/#bit-slice-expressions)\n",
+        "\n",
         "fn user_module(io_in: u8) -> u8 {\n",
         "  adder1(io_in[0:1], io_in[4:5]) as u8\n",
         "}\n",
         "\n",
+        "\n",
+        "// This is a test function that tests whether the functions work as intended\n",
```

Comment:
should we add those comments inline instead?

Reply:
I thought of that but kept it this way because colab doesn't highlight comments, and inline comments looked a bit confusing to me...will keep it this way unless you have a strong preference.